mobile/lib/shared/models/asset.dart

```diff
@@ -35,7 +37,8 @@ class Asset {
         isReadOnly = remote.isReadOnly,
         isOffline = remote.isOffline,
         stackParentId = remote.stackParentId,
-        stackCount = remote.stackCount;
+        stackCount = remote.stackCount,
+        selectedForBackup = BackupSelection.none;
```

Comment:
I'd really like to avoid adding this field to the asset class. Can we store this elsewhere?
E.g. a list of asset IDs with each selection state?